include/slurp.h

```diff
@@ -48,6 +48,8 @@ struct slurp_state {
 		uint32_t choice;
 	} colors;
 
+	char *font_family;
```

Comment:
This can be `const`, to make it clearer that it shouldn't be free'd.

Reply:
Nice, updated my patch accordingly.